lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -263,13 +263,13 @@ public void OnNext(IFailedEvaluator value)
 
             if (value.FailedContexts != null && value.FailedContexts.Count != 0)
             {
-                Logger.Log(Level.Info, "Some active context failed, cannot continue IMRU task");        
```

Comment:
There are some spaces deleted at the end of the line. Given that there are no other changes in this file, we should revert not to create artificial merge issues for other pull requests.